src/ontology/cl-edit.owl

```diff
@@ -26607,11 +26607,14 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "Wikipedia:Intraglomerular_mes
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_1000742 "KUPO:0001032"^^xsd:string)
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "CL:cjm"^^xsd:string) oboInOwl:hasExactSynonym obo:CL_1000742 "intraglomerular mesangial cell"^^xsd:string)
 AnnotationAssertion(oboInOwl:hasOBONamespace obo:CL_1000742 "cell"^^xsd:string)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:9127003"^^xsd:string) Annotation(oboInOwl:hasDbXref "https://doi.org/10.1152/physrev.1998.78.3.723"^^xsd:string) rdfs:comment obo:CL_1000742 "Like smooth muscle, a glomerular mesangial cell is contractile and possesses the filaments actin and myosin. The contractile mechanism is also similar to that of smooth muscle, with Ca2+/calmodulin-activated myosin light-chain kinase instigating myosin-actin cross-bridge cycling. [Stockand, Sansom]. Mesangial cell phagocytosis of apoptotic neutrophils involves a novel CD36-independent, alpha(v)beta3/TSP-mediated mechanism that is uncoupled from chemokine secretion, emphasizing the injury-limiting potential of mesangial cell phagocytosis of apoptotic cells [Hughes et al]."^^xsd:string)
```

Comment:
I think you can remove: [Stockand, Sansom] [Hughes et al] - they should be cited in dbxref anyway.